product/en-US/development_guide/important_log_messages.xml

```diff
@@ -56,4 +56,26 @@
             </para>
         </important>
     </section>
+    <section>
+        <title>Multi cause log message</title>
+        <para>The following table shows some log messages that you may see with an explanation of alternate reasons</para>
+        <informaltable>
+            <tgroup cols="2">
+                <tbody>
+                    <row>
+                        <entry>
+                            <para>INFO  [com.arjuna.ats.arjuna] ObjectStore record was deleted during restoration, users should not deleted records manually: ***/ShadowNoFileLockStore/defaultStore/StateManager/BasicAction/TwoPhaseCoordinator/AtomicAction/SubordinateAtomicAction/JCA/***: java.io.FileNotFoundException: ***/ShadowNoFileLockStore/defaultStore/StateManager/BasicAction/TwoPhaseCoordinator/AtomicAction/SubordinateAtomicAction/JCA/*** (No such file or directory)</para>
+                        </entry>
+                        <entry>
+                            <para>When a transaction is committed at the same instance as a resource adapter/EJB remoting attempts a recover call then you may also see the same message in the log. If you did not manually delete the record it may safely be ignored.</para>
```

Comment:
But this cause is just a harmless race to whoever get in first.

Can we draw out the consequences of each alternate?